### ISSUE TITLE
Hard-coded retrieval of color map and color range for toposeries is a bad idea.

### ISSUE THREAD

**Issue by vladdez** [unfoldtoolbox/UnfoldMakie.jl#82]
i checked 3 cases:

1. ftopo is a Figure
2. ftop is a GridLaout in normal plot 
`plot_topoplotseries!(f[1:2, 1:5], df, Δbin; positions=positions, visual=(label_scatter=false,), layout = (; useColorbar = true)) `
3. ftop is a GridLaout in a combined plot (just look at test_all)

Your strategy was to retrieve colormap and colorrange out of ftopo

1. `d = ftopo.content[1].scene.plots[1]`
2. `d = ftopo.layout.content[2].content.content[1].content.scene.plots[1].plots[1].attributes` 
3. `d = ftopo.layout.content[4].content.content[2].content.scene.plots[1].attributes `--- and number 4 is the order of plot_toposeries in combined function 

So basically in combined figure the number will always change which is kinda a stalemate. And i am not talking that this infinite loop of contents looks unpleasant.
Maybe there is a way to implement it otherwise?

**Comment by behinger** [comment]
https://blog.makie.org/blogposts/v0.19.9-12/

The third point or so shows a solution. Make now has a extract_colormap function. We have to remember to change the makie compat to 0.19.12

**Comment by vladdez** [comment]
resolved
